ironfish/src/rpc/routes/chain/getTransactionStream.ts

```diff
@@ -13,30 +14,66 @@ import { ValidationError } from '../../adapters/errors'
 import { ApiNamespace, router } from '../router'
 
 interface Note {
-  amount: string
+  assetId: string
+  assetName: string
+  value: string
   memo: string
 }
+interface Mint {
+  assetId: string
+  assetName: string
+  value: string
+}
+interface Burn {
+  assetId: string
+  assetName: string
+  value: string
+}
 
 interface Transaction {
   hash: string
   isMinersFee: boolean
   notes: Note[]
+  mints: Mint[]
+  burns: Burn[]
 }
 
 const NoteSchema = yup
   .object()
   .shape({
```

Comment:
I looked around code, this doesn't seem like a breaking change anywhere (other than masp syncer, which will be updated in subsequent PR). Just want to verify this rename from `amount`->`value` will be acceptable. This rename will make all our naming consistent in this rpc.